lib/api/multiObjectDelete.js

```diff
@@ -386,7 +386,7 @@ function multiObjectDelete(authInfo, request, log, callback) {
             return vault.checkPolicies(requestContextParams, authInfo.getArn(),
                 log, (err, authorizationResults) => {
                     // there were no policies so received a blanket AccessDenied
-                    if (err && err.AccessDenied) {
+                    if (err && err.is.AccessDenied) {
```

Comment:
Since `VaultClient` repo has not been updated to use the latest arsenal errors, would this one throw error: `err.is is undefined`? @jbertran @miniscruff

Reply:
I would expect it to. As a stopgap (with a ticket tracking it), you might write `err && err.code === errors.AccessDenied.code`